lib/app-components/addon/components/license-text/styles.scss

```diff
@@ -0,0 +1,3 @@
+.LicenseText {
+    white-space: pre-wrap;
```

Comment:
Might want to add `word-break: break-word;` to prevent breaking words.
<img width="455" alt="Screen Shot 2019-03-28 at 2 22 04 PM" src="https://user-images.githubusercontent.com/4511563/55182938-1fe29300-5165-11e9-9356-10e9fb5a199a.png">